### PR TITLE
Relu

### PR DESCRIPTION
**Pull request by nouiz**
fix gh-4483

### CODE REVIEW

File: doc/library/tensor/basic.txt [hunk 1]
```diff
@@ -676,6 +676,20 @@ Creating Tensor
...
+
+    Returns a tensor filled with 0s of the provided shape.
+
+.. function:: ones(shape, dtype=None)
+
+    :param shape: a tuple/list of scalar with the shape information.
+    :param dtype: the dtype of the new tensor. If None, will use floatX.
+
+    Returns a tensor filled with 1s of the provided shape.
+
```
Comment (f0k) [inline doc/library/tensor/basic.txt]: Was that meant for a different PR?

**Comment by f0k** [comment]
Code and test look good to me.

File: doc/library/tensor/basic.txt (hunk 1 above)
Comment (nouiz) [inline doc/library/tensor/basic.txt]: I have a tendency to add small stuff in PR. Probably not the best, but as travis is slow for our big test suite, it help us.

**Comment by abergeron** [comment]
Yep all good.

**Pull request merged**